src/App.tsx

```diff
@@ -14,11 +15,104 @@ const goodsFromServer: string[] = [
   'Garlic',
 ];
 
-const App: React.FC = () => (
-  <div className="App">
-    <h1>Selected good: -</h1>
-    {goodsFromServer.length}
-  </div>
-);
+class App extends React.Component {
+  state = {
+    selectedGoods: ['Jam'],
+  };
+
+  getHeadline = () => {
+    const { selectedGoods } = this.state;
+
+    if (selectedGoods.length === 0) {
+      return 'No goods selected';
+    }
+
+    if (selectedGoods.length === 1) {
+      return `${selectedGoods[0]} is selected`;
+    }
+
+    const lastEl = selectedGoods[selectedGoods.length - 1];
+    const firstElements = selectedGoods.slice(0, -1).map((item) => {
+      return item;
+    });
+
+    return `${firstElements.join(', ')} and ${lastEl} are selected`;
+  };
+
+  render() {
+    const { selectedGoods } = this.state;
+
+    return (
+      <div className="App">
+        <div className="app-header">
+          <h1>
+            { `${this.getHeadline()}` }
```

Comment:
```suggestion
            {this.getHeadline()}
```
Можна без інтерполяції